src/components/EmptyState/README.md

```diff
@@ -163,8 +163,22 @@ Use to explain a single feature before merchants have used it.
 </EmptyState>
 ```
 
+<!-- content-for: android -->
+
+![Default empty state](/public_images/components/EmptyState/android/default@2x.png)
+
+<!-- /content-for -->
+
+<!-- content-for: ios -->
+
+![Default empty state](/public_images/components/EmptyState/ios/default@2x.png)
+
+<!-- /content-for -->
+
 ### Empty state with subdued footer context
 
+<!-- example-for: web -->
```

Comment:
Not sure if this is meant to close, but otherwise LGTM! 👍

Reply:
Thanks for checking! Confirming: those particular comments don't have a "closing" (just like `<!-- keywords: ... -->`).